### DISCUSSION TITLE
Add Password view 增加密码查看功能

### DISCUSSION THREAD

**Discussion by leezm** [dbeaver/dbeaver#11402] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
When there are many connections，I can’t remember the password for each database.

**Describe the solution you'd like**
Like Chrome or Edge,You can view pass by entering the password of the OS.

**Describe alternatives you've considered**
...

**Additional context**
PLZ!That is necessary!!

**Comment by kseniiaguzeeva** [comment, marked as answer]
It is already available in Enterprise Edition https://dbeaver.com/download/.
![Screenshot 2021-02-18 at 13 19 39](https://user-images.githubusercontent.com/51405061/108342581-0250c380-71ec-11eb-8d79-bea9000ed8a5.png)
